bears/stylus/StylintBear.py

```diff
@@ -0,0 +1,36 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.NpmRequirement import NpmRequirement
+
+
+@linter(executable='stylint',
+        output_format='regex',
+        output_regex=r'(?P<line>\d+):?(?P<column>\d+)?\s+.*'
+                     r'(?P<severity>error|warning)\s+(?P<message>.+)')
+class StylintBear:
+    """
+    Attempts to catch little mistakes (duplication of rules for instance) and
```

Comment:
Choose appropriate ones reflecting as much of stylint's capabilities as possible ;)